python/lsst/pipe/tasks/functors.py

```diff
@@ -918,50 +918,155 @@ def __init__(self,
         self.colCD_2_2 = colCD_2_2
         super().__init__(**kwargs)
 
+    def computeDeltaRaDec(self, x, y, cd11, cd12, cd21, cd22):
+        """Compute the distance on the sphere from x2, y1 to x1, y1.
+
+        Parameters
+        ----------
+        x : `pandas.Series`
+            X pixel coordinate.
+        y : `pandas.Series`
+            Y pixel coordinate.
+        cd11 : `pandas.Series`
+            [1, 1] element of the local Wcs affine transform.
+        cd11 : `pandas.Series`
+            [1, 1] element of the local Wcs affine transform.
+        cd12 : `pandas.Series`
+            [1, 2] element of the local Wcs affine transform.
+        cd21 : `pandas.Series`
+            [2, 1] element of the local Wcs affine transform.
+        cd22 : `pandas.Series`
+            [2, 2] element of the local Wcs affine transform.
+
+        Returns
+        -------
+        raDecTuple : tuple
+            RA and dec conversion of x and y given the local Wcs. Returned
+            units are in radians.
```

Comment:
I think the documentation for tuple returns is to document each tuple element separately, i.e.:
```
Returns
------
ra : `pandas.Series`
    RA coordinate from conversion of (x, y)
dec : ...
```

Also, if the arguments could be scalar floats or `np.ndarray` as well as `pandas.Series`, it's probably best to document that somehow.  Maybe @jonathansick has thoughts on how - there's no single interface type AFAIK, but it's got to be a common pattern.